pkg/server/api/agent/v1/service.go

```diff
@@ -252,6 +261,10 @@ func (s *Service) AttestAgent(stream agentv1.Agent_AttestAgentServer) error {
 	if err := validateAttestAgentParams(params); err != nil {
 		return api.MakeErr(log, codes.InvalidArgument, "malformed param", err)
 	}
+	rpccontext.AddRPCAuditFields(ctx, logrus.Fields{
+		telemetry.NodeAttestorType: params.Data.Type,
+		telemetry.Payload:          api.HashByte(params.Data.Payload),
```

Comment:
it is hashed, so you suggest to just avoid Payload, bundles and tokens?

Reply:
I have some concerns about logging agent attestation payload and join tokens, even if they are hashed. While a hashed value may not be directly usable, it could reveal certain properties about the secret that become useful to an attacker. The agent ID also gets logged on a successful node attestation, so there is at least some traceability in that flow today.